src/Nethermind/Nethermind.Network.Stats/Model/DisconnectReason.cs

```diff
@@ -20,11 +20,6 @@ public enum DisconnectReason : byte
         UnexpectedIdentity = 0x09,
         IdentitySameAsSelf = 0x0a,
         ReceiveMessageTimeout = 0x0b,
-        Other = 0x10,
-        Breach1 = 0x11,
-        Breach2 = 0x12,
-        NdmInvalidHiSignature = 0x13,
-        NdmHostAddressesNotConfigured = 0x14,
-        NdmPeerAddressesNotConfigured = 0x15
+        Other = 0x10
```

Comment:
If I'm not mistaken, for  geth `0x10` is breach of protocol.

Reply:
So... probably fix that also.

Reply:
Yeah its. Its a lot of things afaik

Reply:
Nah, i would stick to the spec

Reply:
We can call it subprotocol but that isnt that clear. Idk